guides/05_tabular-data-science-and-plots/creating-a-dashboard-from-supabase-data.md

```diff
@@ -0,0 +1,122 @@
+# Creating a Dashboard from Supabase Data
+
+Tags: TABULAR, DASHBOARD, PLOTS 
+
+Supabase is a cloud-based open-source backend that provides a PostgreSQL database, authentication, and other useful features for building web and mobile applications. In this tutorial, we will show you how to read data from Supabase plot it in **real-time** on a Gradio Dashboard.
+
+**Prerequisites:** To start, you will need to have a free Supabase account, which you can sign up for here: https://app.supabase.com/
+
+We'll cover the following steps in this end-to-end Guide:
+
+* Creating tables in Supabase
+* Writing data to Supabase using the Supabase Python Client
+* Visualize the data in a real-time dashboard using Gradio
+
+If you already have data on Supabase that you'd like to visualize in a dashboard, you can skip sections and first two sections and go directly to [visualizing the data]()!
```

Comment:
* typo:
> you can skip sections and first two sections and go 

you can skip ~sections and~ *the first two sections and go 

* change 
> [visualizing the data]()

to `[visualizing the data](#visualizing-the-data-in-a-real-time-gradio-dashboard)`